### PR TITLE
Making AOW Workflow more responsive

### PR DESCRIPTION
**Pull request by daniel-samson**
## Description

In order to make more room in workflow conditions and actions, I have removed the labels in the panels.

Desktop
screenshot_20170119_152922

Tablet (landscape) / Small desktop
screenshot_20170119_153125

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by mattlorimer** [comment]
why not just remove them in general

**Comment by daniel-samson** [comment]
What do you mean by in general?

The issue that I had was the label is generated by the parent smarty template. This is due to label being defined in the  metadata.

**Comment by mattlorimer** [comment]
don't remove it by javascript just remove it, via smarty or metadata

File: modules/AOW_Actions/actions/actionCreateRecord.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ function show_crModuleFields(ln){
 
                 //document.getElementById('aow_action_div'+ln).innerHTML ="<select tabindex='116' name='service_vat[]' id='service_vat'>" + cr_fields[ln] + "</select>";
             }
-        }
+        };;;;;;;
```
Comment (mattlorimer) [inline modules/AOW_Actions/actions/actionCreateRecord.js]: semicolon key stuck?

File: modules/AOW_Conditions/conditionLines.js [hunk 2]
```diff
@@ -270,40 +270,45 @@ function insertConditionLine(){
     } else{
         a.innerHTML = condln +1;
     }
-    a.style.width = '5%';
+    a.style.width = ''; // 5%
```
Comment (mattlorimer) [inline modules/AOW_Conditions/conditionLines.js]: do we need to remember it was 5% or even set this if its empty?

File: modules/AOW_WorkFlow/controller.php [hunk 3]
```diff
@@ -277,7 +277,7 @@ protected function action_getActionFieldTypeOptions(){
         }
 
         if($view == 'EditView'){
-            echo "<select type='text' style='width:178px;' name='$aow_field' id='$aow_field' title='' tabindex='116'>". get_select_options_with_id($app_list_strings['aow_action_type_list'], $value) ."</select>";
+            echo "<select type='text' style='' name='$aow_field' id='$aow_field' title='' tabindex='116'>". get_select_options_with_id($app_list_strings['aow_action_type_list'], $value) ."</select>";
```
Comment (mattlorimer) [inline modules/AOW_WorkFlow/controller.php]: do we need empty style

File: themes/SuiteP/css/forms.scss [hunk 4]
```diff
@@ -299,6 +474,150 @@ fieldset[disabled] .btn-primary.active {
...
+  display: block;
+}
+
+#actionLines td[field] label {
+  margin-right: 8px;
+}
+
+#actionLines > tr > td {
+  border: 1px solid $workflow-action-bg;
+  border-top-left-radius: 4px;
```
Comment (mattlorimer) [inline themes/SuiteP/css/forms.scss]: border-radius: 4px;????

File: modules/AOW_Actions/actions/actionCreateRecord.js (hunk 1 above)
Comment (daniel-samson) [inline modules/AOW_Actions/actions/actionCreateRecord.js]: LOL. I just really wanted to make sure that the statement had ended. :D

I have removed it.

File: modules/AOW_Conditions/conditionLines.js (hunk 2 above)
Comment (daniel-samson) [inline modules/AOW_Conditions/conditionLines.js]: Removed lines

File: modules/AOW_WorkFlow/controller.php (hunk 3 above)
Comment (daniel-samson) [inline modules/AOW_WorkFlow/controller.php]: Removed style attribute

File: themes/SuiteP/css/forms.scss (hunk 4 above)
Comment (daniel-samson) [inline themes/SuiteP/css/forms.scss]: It fits the SuiteP Aesthetic.

Comment (mattlorimer) [inline themes/SuiteP/css/forms.scss]: i mean do you need to use border-top-left-radius, border-top-right-radius etc when they are all the same just use border-radius: 4px;

**Comment by samus-aran** [comment]
I too get the jumping down issue as well.  The Email records and general labeling are abit cramp here (see screenshots).
aow1
aow

File: modules/AOW_Conditions/conditionLines.js [hunk 5]
```diff
@@ -270,40 +270,39 @@ function insertConditionLine(){
...
-    e.style.width = '15%';
 
     var f = x.insertCell(5);
     f.id='aow_conditions_fieldInput'+condln;
-    f.style.width = '30%';
 
     condln++;
     condln_count++;
 
+    $('.edit-view-field #conditionLines').find('tbody').last().find('select').change(function () {
```
Comment (samus-aran) [inline modules/AOW_Conditions/conditionLines.js]: Remove and replace with suitable CSS

File: modules/AOW_Actions/actions/actionCreateRecord.js [hunk 6]
```diff
@@ -377,4 +378,18 @@ function date_field_change(field){
         showElem(field + '[2]');
         showElem(field + '[3]');
     }
-}
\ No newline at end of file
+}
+
+var checkActionCreateRecordLines = setInterval(function () {
```
Comment (samus-aran) [inline modules/AOW_Actions/actions/actionCreateRecord.js]: Remove and replace with suitable CSS

**Comment by daniel-samson** [comment]
Updated PR in respect the the review

**Pull request merged**